github-chrome-fullname/index.js

```diff
@@ -1,18 +1,35 @@
 (function() {
     "use strict";
 
+    var allowedSites = [
+        "github.wdf.sap.corp",
+        "huboard.mo.sap.corp"
+    ];
+
     /*global ReplaceRestricter, UserIdStringReplacer, UserIdReplacer*/
     var restricter = new ReplaceRestricter();
     var userIdStringReplacer = new UserIdStringReplacer("https://github.wdf.sap.corp");
     var userIdReplacer = new UserIdReplacer(restricter, userIdStringReplacer);
 
     // Check DOM size every second. After change of DOM elements replace user Ids.
     var lastDomSize;
+
+    var parser = document.createElement('a');
+    parser.href = window.location.href;
+
+    var hostname = parser.hostname;
+    if (allowedSites.indexOf(hostname) === -1) {

```

Comment:
You don't need to do this check here. This check is done via the manifest.
https://github.com/cgrail/github-chrome-fullname/blob/69c4c1283cdd7b7be306f1a9d8bfc1c984c6a725/github-chrome-fullname/manifest.json#L14